setup.py

```diff
@@ -109,6 +155,6 @@ def build_extensions(self):
   keywords = ['astro', 'deblending', 'photometry', 'nmf'],
   ext_modules=ext_modules,
   install_requires=['proxmin>=0.5.0', 'pybind11>=1.7', 'numpy', 'scipy'],
-  cmdclass={'build_ext': BuildExt},
+  cmdclass={'develop': InstallEigen, 'build_ext': BuildExt, 'install': InstallEigen},
```

Comment:
I don't know setuptools well enough to know for certain what this is doing, but you shouldn't have to install the Eigen headers after you've built the module.

Reply:
This is just saying if you use `python setup.py develop` or `python setup.py install`, run the `InstallEigen` class. The order doesn't matter, which is why it's a dict.